libraries/helpers.rb

```diff
@@ -36,35 +36,35 @@ def kafka_init_style
 end
 
 def kafka_init_opts
-  @kafka_init_opts ||= Hash.new.tap do |opts|
+  @kafka_init_opts ||= {}.tap do |opts|
     case kafka_init_style
     when :sysv
-      opts[:env_path] = value_for_platform_family({
+      opts[:env_path] = value_for_platform_family(
         'debian' => '/etc/default/kafka',
-        'default' => '/etc/sysconfig/kafka',
-      })
-      opts[:source] = value_for_platform_family({
+        'default' => '/etc/sysconfig/kafka'
+      )
+      opts[:source] = value_for_platform_family(
         'debian' => 'sysv/debian.erb',
-        'default' => 'sysv/default.erb',
-      })
+        'default' => 'sysv/default.erb'
+      )
       opts[:script_path] = '/etc/init.d/kafka'
       opts[:permissions] = '755'
     when :upstart
       opts[:env_path] = '/etc/default/kafka'
-      opts[:source] = value_for_platform_family({
-        'default' => 'upstart/default.erb',
-      })
+      opts[:source] = value_for_platform_family(
+        'default' => 'upstart/default.erb'
+      )
       opts[:script_path] = '/etc/init/kafka.conf'
       opts[:provider] = ::Chef::Provider::Service::Upstart
       opts[:permissions] = '644'
     when :systemd
-      opts[:env_path] = value_for_platform_family({
+      opts[:env_path] = value_for_platform_family(
         'debian' => '/etc/default/kafka',
-        'default' => '/etc/sysconfig/kafka',
-      })
-      opts[:source] = value_for_platform_family({
+        'default' => '/etc/sysconfig/kafka'
+      )
+      opts[:source] = value_for_platform_family(
         'default' => 'systemd/default.erb'
-      })
+      )
```

Comment:
There are quite some inconsistencies in this method with regards to trailing commas, though `rubocop` doesn't complain when I check this file locally which I think is quite weird?